cli/src/test/java/denominator/cli/DenominatorTest.java

```diff
@@ -89,6 +89,7 @@ public void listsAllProvidersWithCredentials() {
                              "mock       mem:mock                                            false          ",
                              "route53    https://route53.amazonaws.com                       true           accessKey      accessKey secretKey",
                              "route53    https://route53.amazonaws.com                       true           session        accessKey secretKey sessionToken",
+                             "ultradnsrest https://test-restapi.ultradns.com/v2                false          password       username password",
```

Comment:
This is a test class under ultradns-cli module. I think Kushal meant a different place in ultadns-rest module to change this url before releasing, where it is already property driven.